ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkbox/v1/checkbox/_cq_dialog/.content.xml

```diff
@@ -48,24 +48,57 @@
+                                            <type
+                                              wrapperClass="cmp-adaptiveform-checkbox__type"
+                                              jcr:primaryType="nt:unstructured"
+                                              sling:resourceType="granite/ui/components/coral/foundation/form/select"
+                                              sling:orderBefore="visible"
+                                              fieldLabel="Data type of submitted value"
+                                              name="./type"
+                                              key="String"
+                                              value="string[]">
+                                                <items jcr:primaryType="nt:unstructured"
+                                                  granite:class="cmp-adaptiveform-checkbox__type_value">
+                                                    <string jcr:primaryType="nt:unstructured" text="String" value="string"/>
+                                                    <boolean jcr:primaryType="nt:unstructured" text="Boolean" value="boolean"/>
+                                                    <number jcr:primaryType="nt:unstructured" text="Number" value="number"/>
+                                                </items>
+                                            </type>
+                                            <!--<enumOptions jcr:primaryType="nt:unstructured"
+                                              sling:hideResource="{Boolean}true" />-->
+                                            <enums_123 jcr:primaryType="nt:unstructured"
+                                              sling:resourceType="granite/ui/components/coral/foundation/container"
+                                              granite:class="cmp-adaptiveform-checkbox__enums">
+                                                <items jcr:primaryType="nt:unstructured">
+                                                <checkedValue
+                                                  jcr:primaryType="nt:unstructured"
+                                                  sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                                  fieldLabel="When Checked, return value"
+                                                  fieldDescription="Specify the value to be returned when the checkbox is checked by the user."
+                                                  name="./checkedValue"/>
+                                                <enableUncheckedValue
+                                                  jcr:primaryType="nt:unstructured"
+                                                  sling:resourceType="granite/ui/components/coral/foundation/form/switch"
+                                                  wrapperClass="cmp-adaptiveform-checkbox__disable_unchecked_value"
```

Comment:
BEM looks incorrect, it should be `cmp-adaptiveform-checkbox__enable-unchecked-value`

Reply:
Noted